<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SysSetting</class>
 <widget class="QWidget" name="SysSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pbExit">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>400</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>661</width>
     <height>261</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pbParaSet">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设置参数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbTimeCali">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>时间校准</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbParaCali">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>参数校准</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbUpDate">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>系统更新</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbNetSet">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>网络设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbTempSet">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>160</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>温区设置</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lb_Title">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>pbExit</zorder>
  <zorder>lb_Title</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
